JustSaying/HandlerNotRegisteredWithContainerException.cs

```diff
@@ -1,11 +1,25 @@
-﻿using System;
+using System;
+using System.Runtime.Serialization;
 
 namespace JustSaying
 {
     [Serializable]
     public class HandlerNotRegisteredWithContainerException: Exception
     {
-        public HandlerNotRegisteredWithContainerException(string message) : base(message){}
-        public HandlerNotRegisteredWithContainerException(string message, Exception inner) : base(message, inner){}
+        public HandlerNotRegisteredWithContainerException()
```

Comment:
For these new parameterless constructors, we should pass a default message to the base constructor that accepts a string, as that's more meaningful that the message the parameterless constructor of `Exception` will give it.